src/util/address.rs

```diff
@@ -682,14 +689,17 @@ mod tests {
     }
 
     #[test]
-    fn test_bip173_vectors() {
+    fn test_bip173_350_vectors() {
+        // Test vectors valid under both BIP-173 and BIP-350
```

Comment:
I don't think the vectors from `bc1pw508d6qejxtdg4y5r3zarvary0c5xw7kw508d6qejxtdg4y5r3zarvary0c5xw7kt5nd6y` is valid under BIP 173. 

Should this comment only mention BIP-350?

Reply:
Most of them are valid. Bech32m even according to BIP-350 applies only to witness version >= 1 and not 0. So yes, this test case covers both BIPs

Reply:
This is really minor and should not be merge blocking. 

When the comment says, valid under BIP173 and BIP350, the reader would assume _all_ of the vectors are valid not _most_ of them are valid. It is not v0 not being applicable to bech32m, it is about the newer vectors being invalid under previous BIP173(which was defined for future versions too)